src/main/java/core/basesyntax/operations/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.FruitTransaction;
+import core.basesyntax.Storage;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class `BalanceOperation` seems to be in violation of the Single Responsibility Principle (SRP). This class is currently both performing the balance operation and handling the operation. It might be better to separate these responsibilities into different classes. For example, you could have a `BalanceOperation` class that performs the operation and a `OperationHandler` that handles the operation.

Reply:
I have it!